nginx/symfony/config/sites-enabled/www.conf

```diff
@@ -1,32 +1,60 @@
+# This is basically from the docs at http://symfony.com/doc/current/setup/web_server_configuration.html#nginx
+# supplemented with the useful basic configuration from conf.d/basic.conf
+
 server {
     listen 80;
     listen 443 ssl http2;
     server_name localhost;
     root /var/www/web;
 
-    index index.php;
-
     include conf.d/basic.conf;
 
     location / {
-       # This is cool because no php is touched for static content.
+       # Try to serve static files directly, route all others to app.php.
        # include the "?$args" part so non-default permalinks doesn't break when using query string
-       try_files $uri $uri/ /index.php$is_args$args;
+       try_files $uri $uri/ /app.php$is_args$args;
     }
```

Comment:
We should use `sed` and rely on the original prooph nginx www config and replace only the needed lines. We don't need production config, because these images are only for prooph development or as blue print.